index.bs

```diff
@@ -560,7 +568,20 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute returns
 
 NOTE: Most Virtual Reality devices exhibit {{XREnvironmentBlendMode/opaque}} blending behavior. Augmented Reality devices that use transparent optical elements frequently exhibit {{XREnvironmentBlendMode/additive}} blending behavior, and Augmented Reality devices that use passthrough cameras frequently exhibit {{XREnvironmentBlendMode/alpha-blend}} blending behavior.
 
-Each {{XRSession}} has a <dfn for="XRSession">viewer reference space</dfn>, which is an {{XRReferenceSpace}} of type {{XRReferenceSpaceType/viewer}} with an [=identity transform=] [=XRSpace/origin offset=]. The [=XRSession/viewer reference space=] has a <dfn for="XRSession/viewer reference space">list of views</dfn>, which is a [=/list=] of [=view=]s corresponding to the views provided by the [=/XR device=].
+An {{XRSessionMode/inline}} {{XRSession}} is considered a <dfn for="XRSession">direct output session</dfn> if rendering commands performed for the {{XRSession}} are executed in such a way that they are directly displayed on the page, rather than being processed by the [=XR Compositor=].
```

Comment:
Do we really want to be making a statement that a "session" is a direct output session given that this is really a property of the base layer and the base layer can be changed?  If we're treating this PR as a two-way door decision then it seems like we might want to keep these changes encapsulated within the layer spec text.

Reply:
Moved to the `XRRenderState` now, which I agree makes more sense. I also changed it to talk about the render state being in "direct output mode" which sounds more mutable.